### DISCUSSION TITLE
Statuts pour avoir obtenir la liste des formulaires Web d'un individu

### DISCUSSION THREAD

**Discussion by IhsaneRQ** [MTESSDev/FRW#33] [Annonces]
Bonjour,

Y a-t-il d'autres états que ["CRE","REP","MAJ","TRANSMIS"] donnés dans la documentation pour l'attribut "etatsFormulaireRecherche" de l'objet "EntrantRechercherFormulairesSIS" de la méthode "ObtenirFormulairesIndividu"? (Exemple : pour les formulaires en cours )

Aussi, c'est quoi les définitions et descriptions de chaque état de cette liste ["CRE","REP","MAJ","TRANSMIS"] ?

Merci d'avance et bonne journée

**Comment by anisite** [comment]
Bonjour, je viens de pousser une première version de la liste des états ici : [Liste des états](https://github.com/MTESSDev/FRW/blob/main/Documentation/cycle-de-vie-etats.md)

Si ça semble incomplet ou pour autre chose, m'en informer svp

**Reply by IhsaneRQ** [reply]
Bonjour, Parfait.

Merci beaucoup et bonne journée

**Reply by IhsaneRQ** [reply]
Bonjour,

Svp, pour le code d'état EPURE est ce qu'il correspond quand le délai de 30 jours de validation de session est dépassé on va avoir ce code?
Aussi, j'aimerai avoir plus de détail sur le code d'état COURRIEL.

Merci et bonne fin de journée

**Reply by Hutson48** [reply]
L'état EPURE correspond à 90 jours après transmission et 60 jours après inactivité.
L'état COURRIEL correspond à un courriel envoyé pour la reprise du formulaire en mode non identifié.
